ui/src/app/views/workspace-edit/component.ts

```diff
@@ -80,4 +94,28 @@ export class WorkspaceEditComponent implements OnInit {
     this.workspaceCreationError = false;
     this.buttonClicked = false;
   }
+
+  updateWorkspace(): void {
+    if (!this.buttonClicked) {
+      if (isBlank(this.workspace.name)) {
+        this.valueNotEntered = true;
+        const nameArea = document.getElementsByClassName('name-area')[0];
```

Comment:
I think you can do this in a more checked way with `[ngClass]=`. This looks like a relevant example: https://stackoverflow.com/questions/35269179/angular-conditional-class-with-ngclass . My goal is to avoid the controller reaching into the DOM, and instead have the controller publish state variables (like `nameValidationError` or maybe a hierarchy like `this.errors.name`), and then the UI references them.